images/cert-manager-acmesolver/tests/main.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_exec_test" "test" {
+  for_each = toset(fileset(path.module, "*"))
```

Comment:
That's a fun trick...

Reply:
I think this will include the `.tf` file.  We should make this `*.sh`

Reply:
good catch, I was planning on lifting it out into a module, but will fix it here for now